public/content/dungeon-encounters.js

```diff
@@ -3,7 +3,7 @@ import {MonsterRoom, Monster} from '../game/dungeon-rooms.js'
 import {random} from '../game/utils.js'
 
 // Hello. With the imported functions above you can create a dungeon with different rooms and monsters.
-
+//should be able to support even more monsters (4-5)
```

Comment:
Good idea, added a new issue for it https://github.com/oskarrough/slaytheweb/issues/184